ropecli/__init__.py

```diff
@@ -231,7 +232,10 @@ def very_sure(_):
     renamer = rope.refactor.rename.Rename(project, resource, offset)
     changes = renamer.get_changes(new_name, docs=True, unsure=very_sure)
 
-    project.do(changes)
+    if dry:
+        print(changes.get_description())
```

Comment:
Also not sure I like just putting a `print` statement there but I don't know much on how console applications take care of things like this

Reply:
Seems good to me, thanks!